static/app/views/disabledMember/index.tsx

```diff
@@ -0,0 +1,30 @@
+import {RouteComponentProps} from 'react-router';
+
+import NotFound from 'app/components/errors/notFound';
+import HookOrDefault from 'app/components/hookOrDefault';
+import {OrganizationSummary} from 'app/types';
+import withOrganizations from 'app/utils/withOrganizations';
+
+type Props = {
+  organizations: OrganizationSummary[];
+} & RouteComponentProps<{orgId: string}, {}>;
+
+//getsentry will add the view
+const DisabledMemberComponent = HookOrDefault({
```

Comment:
what happens if a user is on this page, then their membership is upgraded? should they be auto redirected back to their org, or should we have a link somewhere?

Reply:
@JoshFerge Good question. Unless we do polling, it will be hard to tell when the subscription has been updated. But what I can do in a follow-up PR is add a redirect from this page if the member is not disabled anymore, good idea.